wsd/COOLWSD.cpp

```diff
@@ -4879,6 +4879,51 @@ class COOLWSDServer
     }
 };
 
+void COOLWSD::processFetchUpdate()
+{
+    try
+    {
+        Poco::URI uriFetch(std::string(INFOBAR_URL));
+        std::shared_ptr<SocketPoll> socketPoll = std::make_shared<SocketPoll>("update");
+        std::shared_ptr<http::Session> sessionFetch = StorageBase::getHttpSession(uriFetch);
+        http::Request request(uriFetch.getPathAndQuery());
+
+        if (!sessionFetch || !socketPoll)
+            return;
+
+        request.add("Accept", "application/json");
+        http::Session::FinishedCallback fetchCallback =
+            [sessionFetch, socketPoll](const std::shared_ptr<http::Session>& httpSession)
+        {
+            const std::shared_ptr<const http::Response> httpResponse = httpSession->response();
```

Comment:
This seems to assume that the POCO config is thread-safe - lets have some locking around this - and ideally not use the poco config to store the information.

Reply:
Yes, it is thread safe